src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -123,6 +123,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     
     <PropertyGroup>
       <FileVersion Condition="'$(FileVersion)' == ''">$(AssemblyVersion)</FileVersion>
+      <InformationalVersion Condition="'$(InformationalVersion)' == ''">$(AssemblyVersion)</InformationalVersion>
```

Comment:
$(Version) will change build-to-build. But $(AssemblyVersion) usually stays the same for an entire release.

Ex $(Version) = 4.0.0-build123
Ex $(AssemblyVersion = 4.0.0.0